WalletWasabi/Nostr/NostrCoordinator.cs

```diff
@@ -0,0 +1,34 @@
+using NBitcoin;
+using WalletWasabi.Exceptions;
+using WalletWasabi.Helpers;
+
+namespace WalletWasabi.Nostr;
+
+public record NostrCoordinator(string Description, string Name, Uri Uri, Network Network)
```

Comment:
This record class should follow the same (implicity) naming convention that all the other configuration classes do, I propose `ApiAnnouncementConfiguration` (see: `DoSConfiguration`, `ServiceConfiguration`, `JsonRpcServerConfiguration` , etc)